src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -28,6 +28,25 @@ typedef NS_ENUM(NSInteger, MTRCommissioningStatus) {
     = 3,
 } API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * A representation of a (vendor, product) pair that identifies a specific product.
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRProductIdentity : NSObject
+
+/**
+ * Controllee's VendorID Attribute of Basic Information cluster
+ */
```

Comment:
```suggestion
```
Because again, depending on where the MTRProductIdentity comes from these may not come from the Basic Information cluster.